src/Helpers/ApartmentHelpers.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace BuildEmpire\Apartment\Helpers;
+
+class ApartmentHelpers
+{
+    /**
+     * Does the schema only contain letters and/or numbers.
+     *
+     * @param $schemaName
+     * @return bool
+     */
+    public static function isSchemaNameValid($schemaName)
+    {
+        return (preg_match('/^[a-z][a-z0-9_]*$/', $schemaName) && !ctype_upper($schemaName));
```

Comment:
`ctype_upper` isn't needed as the regular expression checks exclusively for lowercase, 0-9 & `_`.